Cargo.toml

```diff
@@ -13,10 +13,13 @@ anyhow = "1.0.34"
 async-trait = "0.1.42"
 futures = "0.3.8"
 newtype_derive = "0.1.6"
-petgraph = "0.6.0"
 serde_json = "1.0"
 thiserror = "1.0"
 
+[dependencies.petgraph]
+version = "0.6.2"
+features =  [ "serde-1" ]
```

Comment:
I haven't taken a look at the serialized form. And yes, I was thinking also that it may not matter with the upgrade plan we discussed. That really helps with quite a few issues!

Reply:
It looks like they make some effort to keep this stable: https://docs.rs/petgraph/latest/src/petgraph/graph_impl/serialization.rs.html#17-50